<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultWindow</class>
 <widget class="QMainWindow" name="ResultWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>686</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>686</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ResultWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MplSoilMoisture" name="MplSoilMoisture" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>550</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>550</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>550</width>
      <height>300</height>
     </size>
    </property>
   </widget>
   <widget class="MplHumidity" name="MplHumidity" native="true">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>550</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>550</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>300</height>
     </size>
    </property>
   </widget>
   <widget class="MplTemperature" name="MplTemperature" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>550</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>550</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>300</height>
     </size>
    </property>
   </widget>
   <widget class="MplValveOpen" name="MplValveOpen" native="true">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>350</y>
      <width>550</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>550</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>300</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>160</y>
      <width>31</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>hot.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>160</y>
      <width>161</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Temperature:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Temperature">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>200</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>380</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Valve Opening:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>380</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>irrigation.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ValveOpen">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>420</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>40</y>
      <width>41</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>plant.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SoilMoisture">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>40</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Soil Moisture:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>100</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Humidity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>100</y>
      <width>41</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>humidity.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Humidity">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>9</y>
      <width>631</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MEMBERSHIP FUNCTION AND OUTPUT PLOT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>INPUT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>339</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>OUTPUT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>600</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#dc3545;
color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>exit.png</normaloff>exit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplSoilMoisture</class>
   <extends>QWidget</extends>
   <header>mplsoilmoisture.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MplHumidity</class>
   <extends>QWidget</extends>
   <header>mplhumidity.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MplTemperature</class>
   <extends>QWidget</extends>
   <header>mpltemperature.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MplValveOpen</class>
   <extends>QWidget</extends>
   <header>mplvalveopen.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
